### PR TITLE
fix: reduce flickering of images of profile/source images

### PR DESCRIPTION
**Pull request by omBratteng**
- Reduces flickering of profile and source images in sidebar and header
- Removes flickering of sources on source page

AS-355

### Preview domain
https://as-355-fix-image-flash.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/MainLayout.tsx [hunk 1]
```diff
@@ -200,7 +177,22 @@ function MainLayoutComponent({
...
+            enableSearch={enableSearch}
+            activePage={activePage}
+            showDnd={showDnd}
+            dndActive={dndActive}
+            isNavButtons={isNavItemsButton}
+            onShowDndClick={onShowDndClick}
+            onLogoClick={onLogoClick}
+            setOpenMobileSidebar={() => onMobileSidebarToggle(false)}
+          />
+        )}
```
Comment (omBratteng) [inline packages/shared/src/components/MainLayout.tsx:195]: I removed the `sidebarRendered === null` check, as `useSidebarRendered` either returns `true` or `false`. And I moved it down here, as something with it being in a const made it re-render often

File: packages/webapp/components/common.tsx [hunk 2]
```diff
@@ -70,15 +76,9 @@ export const TopList = ({
...
   const isMobile = useViewSize(ViewSize.MobileL);
-  const MobileDiv = classed(
-    'div',
-    'flex flex-col border-b border-b-border-subtlest-tertiary p-4',
-    className,
-  );
-  const CardElement = classed(Card, '!p-4 !max-h-none', className);
-  const Wrapper = isMobile ? MobileDiv : CardElement;
+  const Wrapper = isMobile ? TopListMobileDiv : TopListCardElement;
   return (
```
Comment (omBratteng) [inline packages/webapp/components/common.tsx:80]: Option was either memoize these or move outside, moving outside was easiest

**Review by rebelchris — APPROVED** [review]
Maybe that sidebar issue was the main one.
Hoping it will solve the shortcuts as well, so let's give it a try

**Review by nensidosari — APPROVED** [review]

**Review by sshanzel — APPROVED** [review]
Looks great! 🚀

**Pull request merged**